### ISSUE TITLE
Unable to find VV Path

### ISSUE THREAD

**Issue by JasonTheAdams** [bradp/vv#143]
I installed homebrew and did the subsequent instructions for installing vv. Unfortunately, when I try `vv` anything, it asks for the VVV installation directory. I tried pointing it to /usr/local/bin/vagrant vv as well as the path provided by `homebrew ls vv`, but everything returns `[Error] Path specified is not a VVV root directory. Where is VVV installed?`.

Am I missing something? Thanks!

**Comment by bradp** [comment]
Do you have VVV installed?

**Comment by JasonTheAdams** [comment]
.....That was one of my denser moments. Let us never speak of this. :D

**Comment by thinkhuman** [comment]
I'm now getting the same error. Here's my .vv-config--and it's where the Vagrantfile is located:

` "path": "/Users/james/vagrant-local" }`

I can see vvv.dev, and use an existing vv-created vm. But why am I getting this error?

**Comment by Marco-G** [comment]
Same as @thinkhuman here.

```
Your version of vv ( 1.9.3 )  does not match the current Github version ( 1.10.2 ). Please update with vv --update 
[Error] Path specified is not a VVV root directory. Where is VVV installed?
```

**Comment by bradp** [comment]
@Marco-G You're out of date. Please update your copy of `vv`, this bug has been fixed in 1.10.

**Comment by thinkhuman** [comment]
@Marco-G @bradp I never got update to work properly, so I blew away the ```vv``` install and reinstalled. Then, it picked up the latest version.

**Comment by JasonTheAdams** [comment]
@bradp When I do `vv --update` it does the process but leaves me at 1.9.3.

**Comment by Marco-G** [comment]
Yep, same here... vv-1.9.3 already installed 
and nothing happens.

**Comment by bradp** [comment]
As the [README](https://github.com/bradp/vv#updating) states, if `vv --update` does not work, you can do `vv --force-update`.

**Comment by thinkhuman** [comment]
@bradp ```--force-update``` definitely didn't work. Same result, promise.

**Comment by JasonTheAdams** [comment]
Here's what I'm getting:
```
Your version of vv ( 1.9.3 )  does not match the current Github version ( 1.10.2 ). Please update with vv --update
Forcing boostrapped update...
/usr/local/bin/vv: line 320: /usr/local/bin/vv: Permission denied
rm: /Users/jason/vv-1.10.2: is a directory
[Success] vv has been updated to the latest version. Thanks!
```
Afterwards it's still at 1.9.3. What's interesting is I checked /usr/local/bin/vv and all the permissions are such that this current user shouldn't have any permissions issues.

**Comment by johnkvang** [comment]
Hey guys, was there ever a fix to this?  I seem to be stuck in 1.9.3. I've tried both update options and no luck.
